jabref-template/listrefs.end.layout

```diff
@@ -95,7 +95,9 @@
     var n_publications=0;
     for (var i=first_year; i<=last_year; ++i)
       n_publications += table[i-first_year+1][1];
-    document.getElementById("n_publications").innerHTML = "Total number of publications: <b>" + n_publications + "</b>.";
+    document.getElementById("n_publications").innerHTML =
+        "Known publications using deal.II. Total: <b>" + n_publications +
+        "</b>. See <a href=\"#list\">below for individual publications</a>.";
```

Comment:
this ``\"`` might also freak out JabRef.

Reply:
can you try single quotes to see if that fixes the problem? Otherwise move the text outside javascript back into the body.

Reply:
Hm, let me take this out then.